source/code-generation/query-converter/enable-query-converter.txt

```diff
@@ -36,11 +28,27 @@ Steps
 
    .. step:: Read the user agreement
 
+   .. step:: (Optional) Review telemetry preferences
+
+      Telemetry data about the source and converted queries are 
+      temporarily stored by MongoDB to improve the query converter 
+      feature. You can disable sending telemetry data by unchecking the 
+      :guilabel:`Allow MongoDB to temporarily store source and coverted 
+      queries to improve the product feature`
+      check box.
+
+      .. note::
+
+         You can also disable telemetry data by toggling the 
+         :guilabel:`Allow queries to be stored temporarily` 
+         switch in global project settings.
+
    .. step:: Enable the query converter feature
 
-      Click :guilabel:`Use Query Converter`. 
+      Click :guilabel:`Start Using Query Converter`. 
```

Comment:
I think based on your screenshots yesterday this button text has changed in the next release:
![Screenshot 2024-02-22 at 10 12 05 AM](https://github.com/mongodb/docs-relational-migrator/assets/85948430/2b61a1aa-4754-47e3-861e-9685c929ed41)